app/controllers/whitehall_media_controller.rb

```diff
@@ -1,6 +1,7 @@
 class WhitehallMediaController < BaseMediaController
   def download
-    path = "/government/uploads/#{params[:path]}.#{params[:format]}"
+    path = "/government/uploads/#{params[:path]}"
+    path += ".#{params[:format]}" if params[:format].present?
```

Comment:
Minor: I might be tempted to use [Pathname#sub_ext](https://ruby-doc.org/stdlib-2.4.0/libdoc/pathname/rdoc/Pathname.html#method-i-sub_ext) to perform this kind of path manipulation, but I don't think it makes much difference.

Reply:
Oh. That's interesting. I haven't come across that method before. Thanks!